src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
Maybe programmers in google have guidelines and have to write ++i, for me it's still not that important. This issue is just bikeshedding.

and chrome is compiled using clang, qtox using g++, if you think this is important and introduces regression please compare assembly of both pre and post increments using both compilers and list advances to convince me it's actually worth writing ++i.